intellij/org.eclipse.xtend.idea/src/org/eclipse/xtend/core/idea/completion/XtendCompletionContributor.xtend

```diff
@@ -58,7 +56,7 @@ class XtendCompletionContributor extends AbstractXtendCompletionContributor {
 	protected def completeAbstractSuperMethods() {
 		extend(
 			CompletionType.BASIC,
-			psiElement.withEReference(XtendPackage.Literals.XTEND_TYPE_DECLARATION__MEMBERS)
+			grammarAccess.memberAccess.newKeyword_2_2_2
```

Comment:
I changed it to use the feature. As it is just computing the elements once on startup, I don't think it is a problem performance-wise. 
Changing it revealed a serious bug, which is fixed in the updated commit. thx.